tests/sklearn/test_sklearn_autolog.py

```diff
@@ -863,8 +863,9 @@ def test_autolog_does_not_throw_when_mlflow_logging_fails(func_to_fail):
         mock_func.assert_called_once()
 
 
+@pytest.mark.parametrize("_", range(100))
```

Comment:
Run `test_autolog_logs_signature_and_input_example` 100 times to ensure the failure is flaky